### DISCUSSION TITLE
Set mowing mode?

### DISCUSSION THREAD

**Discussion by Nusse469** [mikey0000/Mammotion-HA#72] [General]
First great job on this integration! I use it almost daily and for my basic needs, it has worked very well.

My question is, can I set a mowing mode? For example, once a week I would like to tell the Luba to mow the perimeter and I can do that via the built in task, but would like to keep it all in one if I can within Home Assistant.  I currently use Node Red to kick off the bot with your integration and I imagine I can kick the mode via the data entry but am unsure of what format (or even in it is supported) your integration requires to pass that to the Luba.

Thanks.

**Comment by mikey0000** [comment]
Glad you appreciate it, Yeah that's the end goal, it can be done, just doing a lot of the foundation work right now. Instructions on this will come once the configuration options are complete
